src/CodeGen_ARM.cpp

```diff
@@ -462,13 +657,30 @@ void CodeGen_ARM::visit(const Mul *op) {
 }
 
 void CodeGen_ARM::visit(const Div *op) {
+#if 0
+    // TODO: This is probably the easiest thing to convert to IRMatch rewriting, but it
+    // needs work.
```

Comment:
Not a blocker, but what sort of work does it need?

Reply:
Removed, this was to show @abadams WIP future possibilities